scripts/jsDoc.mjs

```diff
@@ -0,0 +1,19 @@
+/**
+ * Build script to generate markdown files from JSDoc comments in source
```

Comment:
**Explanation**

1. reads all the `.js` files in `src/formatters`
2. renders a markdown string from structured code comments in those files
3. writes the markdown file to `dist/docs/` so it can be used by storybook